labgrid/driver/jlinkdriver.py

```diff
@@ -0,0 +1,43 @@
+from importlib import import_module
+import socket
+import attr
+
+from ..factory import target_factory
+from ..resource.remote import NetworkJLinkDevice
+from ..util.proxy import proxymanager
+from .common import Driver
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class JLinkDriver(Driver):
+    bindings = {"jlink_device": {"JLinkDevice", "NetworkJLinkDevice"}, }
```

Comment:
Maybe `s/jlink_device/device/`? Since we know that we are in the jlink context anyways.